docs/course_materials/exercises/05_Control_structures/homework_loop.md

```diff
@@ -0,0 +1,21 @@
+# Control Structures - Homework exercise 
+
+## for loop
+
+### ten.py
+Create a file *ten.py*
+
+Write a program to print numbers from 1 to 10.
+
+### natural.py
+Create a file *natural.py*
+
```

Comment:
Replace the order of the lines:
new line
Create a file...

Reply:
done